tests/test_default_qubit.py

```diff
@@ -366,7 +366,7 @@ def test_apply_errors(self):
             p = [np.array([1, 0, 1, 1, 1])/np.sqrt(3)]
             self.dev.apply('QubitStateVector', wires=[0, 1], par=[p])
 
-        with self.assertRaisesRegex(ValueError, "BasisState parameter must be an array of 0/1 integers"):
+        with self.assertRaisesRegex(ValueError, "BasisState parameter must be an array of len\(wires\) many 0/1 integers\."):
```

Comment:
Any reason for escaping these conventional characters here?

Reply:
I would suggest matching
```python3
"BasisState parameter must be an length-{} array of 0 or 1 integers.".format(2**len(wires))
```
here as well

Reply:
I will use "be a length-{}"